ui/admin/form.php

```diff
@@ -555,23 +551,18 @@
 									 * @since 2.4.1
 									 */
 									do_action( 'pods_act_editor_before_metabox', $pod );
-									?>
-									<div id="pods-meta-box-<?php echo esc_attr( sanitize_title( $group['label'] ) ); ?>" class="postbox">
-										<div class="handlediv" title="Click to toggle"><br /></div>
-										<h3 class="hndle">
-								<span>
-									<?php
+
 									if ( ! $more && 1 == count( $groups ) ) {
 										$title = __( 'Fields', 'pods' );
 									} else {
 										$title = $group['label'];
 									}
 
 									/** This filter is documented in classes/PodsMeta.php */
-									echo apply_filters( 'pods_meta_default_box_title', $title, $pod, $fields, $pod->api->pod_data['type'], $pod->pod );
+									$title = apply_filters( 'pods_meta_default_box_title', $title, $pod, $fields, $pod->api->pod_data['type'], $pod->pod );
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $title

<sub>hash: 70fc25a3711d14f5817cbd31799d7a8b</sub>